### PR TITLE
Fix fix add test for slider

### PR DESCRIPTION
**Pull request by ZanBin**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

File: src/components/Slider/Slider.tsx [hunk 1]
```diff
@@ -235,21 +239,22 @@ const Slider: SliderComponentProps = ({
    */
   const animatedTrackStyle = useAnimatedStyle(
     () => ({
-      width: withTiming(progress.value, {duration: 1}),
+      width: withTiming(progress.value, { duration: 1 }),
     }),
-    [progress],
+    [progress, value],
```
Comment (loc-nguyenthien) [inline src/components/Slider/Slider.tsx:244]: Why we need to add this value?
